### PR TITLE
Adding docker support

### PR DESCRIPTION
**Pull request by aliabbasjaffri**
Added docker support to run the application on any architecture.

### CODE REVIEW

**Comment by origranot** [comment]
Can you change it according to the nest.js migration?

**Comment by aliabbasjaffri** [comment]
Updated!

File: README.md [hunk 1]
```diff
@@ -36,6 +38,7 @@ cd url-shortener-1
 Step 4. Install the node dependencies.
 ```sh
 npm install
+node app.js
```
Comment (origranot) [inline README.md:41]: Why is that?

**Comment by origranot** [comment]
@aliabbasjaffri Hey!

I just review the changes. please fix the comments.

When you run nestjs you will do it with npm start (locally also)

File: README.md (hunk 1 above)
Comment (aliabbasjaffri) [reply README.md]: Seems like a merge error. I just fixed it and pushed the update

**Pull request merged**